### ISSUE TITLE
Script Exception: No module named ordered_dict

### ISSUE THREAD

**Issue by magichome** [xebialabs-community/xlr-xlrelease-plugin#42]
Under XLR 9.7.2 and attempting to test connection to localhost:5516, I get...

`Caused by: javax.script.ScriptException: ImportError: No module named ordered_dict in <script> at line number 12`

**Comment by rubenryhan** [comment]
@magichome  have you found any workaround/solution for this issue? I am facing same issue with this particular plugin, as well as, [xlr-ansible-tower-plugin](https://github.com/xebialabs-community/xlr-ansible-tower-plugin) , on XLR version 9.7.6 and 9.8.0. 

regards

**Comment by rubenryhan** [comment]
Hi team,
I have worked out what the issue was in this case, it is incompatibility between urllib3 and requests module. I have not done extensive testing yet, but here is what I have so far - 

`1.15.1`  <<< version of urllib3 in XLR v9.7.2 (working environment with xlr-xlrelease-plugin installed)
`1.25.9`  <<< version of urllib3 in XLR v9.8.0/9.7.6 (issue occurs in this environment with xlr-xlrelease-plugin installed)

`2.19.1`  <<< version of requests module shipped with xlr-xlrelease-plugin, according to documentations that are around urllib3 is only supported up to v1.23.x for requests module 2.19.1 

In my case, I have replaced the requests module within xlr-xlrelease-plugin-2.3.6.jar file with requests module (v 2.24.0) , and that has resolved the issue across the board with ImportError: No module named ordered_dict. Also, I have tested removing requests module from xlr-xlrelease-plugin-2.3.7.jar file and that also resolves the issue.

regards,
